<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>visualization</class>
 <widget class="QDialog" name="visualization">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>982</width>
    <height>556</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <property name="styleSheet">
   <string notr="true">
#visualization{ 
border-image: url(:/pic/15f22a31ece69881bd29efa5e524b999.jpeg);
}</string>
  </property>
  <widget class="QTextBrowser" name="textBrowser">
   <property name="geometry">
    <rect>
     <x>630</x>
     <y>50</y>
     <width>321</width>
     <height>471</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgba(255, 255, 255, 155);</string>
   </property>
   <property name="html">
    <string>&lt;!DOCTYPE HTML PUBLIC &quot;-//W3C//DTD HTML 4.0//EN&quot; &quot;http://www.w3.org/TR/REC-html40/strict.dtd&quot;&gt;
&lt;html&gt;&lt;head&gt;&lt;meta name=&quot;qrichtext&quot; content=&quot;1&quot; /&gt;&lt;style type=&quot;text/css&quot;&gt;
p, li { white-space: pre-wrap; }
&lt;/style&gt;&lt;/head&gt;&lt;body style=&quot; font-family:'SimSun'; font-size:9pt; font-weight:400; font-style:normal;&quot;&gt;
&lt;p style=&quot; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;&quot;&gt;&lt;span style=&quot; color:#808000;&quot;&gt;void&lt;/span&gt;&lt;span style=&quot; color:#c0c0c0;&quot;&gt; &lt;/span&gt;left_move()&lt;/p&gt;
&lt;p style=&quot; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;&quot;&gt;&lt;span style=&quot; font-family:'Courier New';&quot;&gt;{&lt;/span&gt;&lt;/p&gt;
&lt;p style=&quot; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;&quot;&gt;&lt;span style=&quot; font-family:'Courier New'; color:#c0c0c0;&quot;&gt;    &lt;/span&gt;&lt;span style=&quot; font-family:'Courier New';&quot;&gt;s.remove(p,&lt;/span&gt;&lt;span style=&quot; font-family:'Courier New'; color:#c0c0c0;&quot;&gt; &lt;/span&gt;&lt;span style=&quot; font-family:'Courier New'; color:#000080;&quot;&gt;1&lt;/span&gt;&lt;span style=&quot; font-family:'Courier New';&quot;&gt;);&lt;/span&gt;&lt;/p&gt;
&lt;p style=&quot; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;&quot;&gt;&lt;span style=&quot; font-family:'Courier New'; color:#c0c0c0;&quot;&gt;    &lt;/span&gt;&lt;span style=&quot; font-family:'Courier New';&quot;&gt;s.insert(--p,&lt;/span&gt;&lt;span style=&quot; font-family:'Courier New'; color:#c0c0c0;&quot;&gt; &lt;/span&gt;&lt;span style=&quot; font-family:'Courier New'; color:#008000;&quot;&gt;'.'&lt;/span&gt;&lt;span style=&quot; font-family:'Courier New';&quot;&gt;);&lt;/span&gt;&lt;/p&gt;
&lt;p style=&quot; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;&quot;&gt;&lt;span style=&quot; font-family:'Courier New'; color:#c0c0c0;&quot;&gt;    &lt;/span&gt;&lt;span style=&quot; font-family:'Courier New'; color:#808000;&quot;&gt;if&lt;/span&gt;&lt;span style=&quot; font-family:'Courier New';&quot;&gt;(!p)&lt;/span&gt;&lt;/p&gt;
&lt;p style=&quot; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;&quot;&gt;&lt;span style=&quot; font-family:'Courier New'; color:#c0c0c0;&quot;&gt;    &lt;/span&gt;&lt;span style=&quot; font-family:'Courier New';&quot;&gt;{&lt;/span&gt;&lt;/p&gt;
&lt;p style=&quot; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;&quot;&gt;&lt;span style=&quot; font-family:'Courier New'; color:#c0c0c0;&quot;&gt;        &lt;/span&gt;&lt;span style=&quot; font-family:'Courier New';&quot;&gt;s.insert(p++,&lt;/span&gt;&lt;span style=&quot; font-family:'Courier New'; color:#c0c0c0;&quot;&gt; &lt;/span&gt;&lt;span style=&quot; font-family:'Courier New'; color:#008000;&quot;&gt;'0'&lt;/span&gt;&lt;span style=&quot; font-family:'Courier New';&quot;&gt;);&lt;/span&gt;&lt;/p&gt;
&lt;p style=&quot; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;&quot;&gt;&lt;span style=&quot; font-family:'Courier New'; color:#c0c0c0;&quot;&gt;    &lt;/span&gt;&lt;span style=&quot; font-family:'Courier New';&quot;&gt;}&lt;/span&gt;&lt;/p&gt;
&lt;p style=&quot; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;&quot;&gt;&lt;span style=&quot; font-family:'Courier New'; color:#c0c0c0;&quot;&gt;    &lt;/span&gt;&lt;span style=&quot; font-family:'Courier New'; color:#808000;&quot;&gt;if&lt;/span&gt;&lt;span style=&quot; font-family:'Courier New';&quot;&gt;(p&lt;/span&gt;&lt;span style=&quot; font-family:'Courier New'; color:#c0c0c0;&quot;&gt; &lt;/span&gt;&lt;span style=&quot; font-family:'Courier New';&quot;&gt;&amp;gt;=&lt;/span&gt;&lt;span style=&quot; font-family:'Courier New'; color:#c0c0c0;&quot;&gt; &lt;/span&gt;&lt;span style=&quot; font-family:'Courier New';&quot;&gt;FE_s.length()-&lt;/span&gt;&lt;span style=&quot; font-family:'Courier New'; color:#000080;&quot;&gt;2&lt;/span&gt;&lt;span style=&quot; font-family:'Courier New';&quot;&gt;)&lt;/span&gt;&lt;/p&gt;
&lt;p style=&quot; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;&quot;&gt;&lt;span style=&quot; font-family:'Courier New'; color:#c0c0c0;&quot;&gt;    &lt;/span&gt;&lt;span style=&quot; font-family:'Courier New';&quot;&gt;{&lt;/span&gt;&lt;/p&gt;
&lt;p style=&quot; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;&quot;&gt;&lt;span style=&quot; font-family:'Courier New'; color:#c0c0c0;&quot;&gt;        &lt;/span&gt;&lt;span style=&quot; font-family:'Courier New';&quot;&gt;s.chop(&lt;/span&gt;&lt;span style=&quot; font-family:'Courier New'; color:#000080;&quot;&gt;1&lt;/span&gt;&lt;span style=&quot; font-family:'Courier New';&quot;&gt;);&lt;/span&gt;&lt;/p&gt;
&lt;p style=&quot; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;&quot;&gt;&lt;span style=&quot; font-family:'Courier New'; color:#c0c0c0;&quot;&gt;    &lt;/span&gt;&lt;span style=&quot; font-family:'Courier New';&quot;&gt;}&lt;/span&gt;&lt;/p&gt;
&lt;p style=&quot; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;&quot;&gt;&lt;span style=&quot; font-family:'Courier New';&quot;&gt;}&lt;/span&gt;&lt;/p&gt;
&lt;p style=&quot; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;&quot;&gt;&lt;span style=&quot; font-family:'Courier New'; color:#808000;&quot;&gt;void&lt;/span&gt;&lt;span style=&quot; font-family:'Courier New'; color:#c0c0c0;&quot;&gt; &lt;/span&gt;&lt;span style=&quot; font-family:'Courier New';&quot;&gt;right_move()&lt;/span&gt;&lt;/p&gt;
&lt;p style=&quot; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;&quot;&gt;&lt;span style=&quot; font-family:'Courier New';&quot;&gt;{&lt;/span&gt;&lt;/p&gt;
&lt;p style=&quot; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;&quot;&gt;&lt;span style=&quot; font-family:'Courier New'; color:#c0c0c0;&quot;&gt;    &lt;/span&gt;&lt;span style=&quot; font-family:'Courier New';&quot;&gt;s.remove(p,&lt;/span&gt;&lt;span style=&quot; font-family:'Courier New'; color:#c0c0c0;&quot;&gt; &lt;/span&gt;&lt;span style=&quot; font-family:'Courier New'; color:#000080;&quot;&gt;1&lt;/span&gt;&lt;span style=&quot; font-family:'Courier New';&quot;&gt;);&lt;/span&gt;&lt;/p&gt;
&lt;p style=&quot; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;&quot;&gt;&lt;span style=&quot; font-family:'Courier New'; color:#c0c0c0;&quot;&gt;    &lt;/span&gt;&lt;span style=&quot; font-family:'Courier New';&quot;&gt;s.insert(++p,&lt;/span&gt;&lt;span style=&quot; font-family:'Courier New'; color:#c0c0c0;&quot;&gt; &lt;/span&gt;&lt;span style=&quot; font-family:'Courier New'; color:#008000;&quot;&gt;'.'&lt;/span&gt;&lt;span style=&quot; font-family:'Courier New';&quot;&gt;);&lt;/span&gt;&lt;/p&gt;
&lt;p style=&quot; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;&quot;&gt;&lt;span style=&quot; font-family:'Courier New'; color:#c0c0c0;&quot;&gt;    &lt;/span&gt;&lt;span style=&quot; font-family:'Courier New'; color:#808000;&quot;&gt;if&lt;/span&gt;&lt;span style=&quot; font-family:'Courier New';&quot;&gt;(s[&lt;/span&gt;&lt;span style=&quot; font-family:'Courier New'; color:#000080;&quot;&gt;0&lt;/span&gt;&lt;span style=&quot; font-family:'Courier New';&quot;&gt;]&lt;/span&gt;&lt;span style=&quot; font-family:'Courier New'; color:#c0c0c0;&quot;&gt; &lt;/span&gt;&lt;span style=&quot; font-family:'Courier New';&quot;&gt;==&lt;/span&gt;&lt;span style=&quot; font-family:'Courier New'; color:#c0c0c0;&quot;&gt; &lt;/span&gt;&lt;span style=&quot; font-family:'Courier New'; color:#008000;&quot;&gt;'0'&lt;/span&gt;&lt;span style=&quot; font-family:'Courier New';&quot;&gt;)&lt;/span&gt;&lt;/p&gt;
&lt;p style=&quot; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;&quot;&gt;&lt;span style=&quot; font-family:'Courier New'; color:#c0c0c0;&quot;&gt;    &lt;/span&gt;&lt;span style=&quot; font-family:'Courier New';&quot;&gt;{&lt;/span&gt;&lt;/p&gt;
&lt;p style=&quot; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;&quot;&gt;&lt;span style=&quot; font-family:'Courier New'; color:#c0c0c0;&quot;&gt;        &lt;/span&gt;&lt;span style=&quot; font-family:'Courier New';&quot;&gt;s.remove(&lt;/span&gt;&lt;span style=&quot; font-family:'Courier New'; color:#000080;&quot;&gt;0&lt;/span&gt;&lt;span style=&quot; font-family:'Courier New';&quot;&gt;,&lt;/span&gt;&lt;span style=&quot; font-family:'Courier New'; color:#c0c0c0;&quot;&gt; &lt;/span&gt;&lt;span style=&quot; font-family:'Courier New'; color:#000080;&quot;&gt;1&lt;/span&gt;&lt;span style=&quot; font-family:'Courier New';&quot;&gt;);&lt;/span&gt;&lt;/p&gt;
&lt;p style=&quot; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;&quot;&gt;&lt;span style=&quot; font-family:'Courier New'; color:#c0c0c0;&quot;&gt;        &lt;/span&gt;&lt;span style=&quot; font-family:'Courier New';&quot;&gt;--p;&lt;/span&gt;&lt;/p&gt;
&lt;p style=&quot; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;&quot;&gt;&lt;span style=&quot; font-family:'Courier New'; color:#c0c0c0;&quot;&gt;    &lt;/span&gt;&lt;span style=&quot; font-family:'Courier New';&quot;&gt;}&lt;/span&gt;&lt;/p&gt;
&lt;p style=&quot; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;&quot;&gt;&lt;span style=&quot; font-family:'Courier New'; color:#c0c0c0;&quot;&gt;    &lt;/span&gt;&lt;span style=&quot; font-family:'Courier New'; color:#808000;&quot;&gt;if&lt;/span&gt;&lt;span style=&quot; font-family:'Courier New';&quot;&gt;(p&lt;/span&gt;&lt;span style=&quot; font-family:'Courier New'; color:#c0c0c0;&quot;&gt; &lt;/span&gt;&lt;span style=&quot; font-family:'Courier New';&quot;&gt;&amp;gt;=&lt;/span&gt;&lt;span style=&quot; font-family:'Courier New'; color:#c0c0c0;&quot;&gt; &lt;/span&gt;&lt;span style=&quot; font-family:'Courier New';&quot;&gt;FE_s.length()-&lt;/span&gt;&lt;span style=&quot; font-family:'Courier New'; color:#000080;&quot;&gt;1&lt;/span&gt;&lt;span style=&quot; font-family:'Courier New';&quot;&gt;)&lt;/span&gt;&lt;/p&gt;
&lt;p style=&quot; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;&quot;&gt;&lt;span style=&quot; font-family:'Courier New'; color:#c0c0c0;&quot;&gt;    &lt;/span&gt;&lt;span style=&quot; font-family:'Courier New';&quot;&gt;{&lt;/span&gt;&lt;/p&gt;
&lt;p style=&quot; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;&quot;&gt;&lt;span style=&quot; font-family:'Courier New'; color:#c0c0c0;&quot;&gt;        &lt;/span&gt;&lt;span style=&quot; font-family:'Courier New';&quot;&gt;s.append(&lt;/span&gt;&lt;span style=&quot; font-family:'Courier New'; color:#008000;&quot;&gt;'0'&lt;/span&gt;&lt;span style=&quot; font-family:'Courier New';&quot;&gt;);&lt;/span&gt;&lt;/p&gt;
&lt;p style=&quot; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;&quot;&gt;&lt;span style=&quot; font-family:'Courier New'; color:#c0c0c0;&quot;&gt;    &lt;/span&gt;&lt;span style=&quot; font-family:'Courier New';&quot;&gt;}&lt;/span&gt;&lt;/p&gt;
&lt;p style=&quot; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;&quot;&gt;&lt;span style=&quot; font-family:'Courier New';&quot;&gt;}&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
   </property>
  </widget>
  <widget class="QPushButton" name="AutoButton">
   <property name="geometry">
    <rect>
     <x>300</x>
     <y>490</y>
     <width>111</width>
     <height>41</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgba(255, 255, 255, 155);
font: 10pt &quot;黑体&quot;;</string>
   </property>
   <property name="text">
    <string>自动进行</string>
   </property>
  </widget>
  <widget class="QPushButton" name="NextButton">
   <property name="enabled">
    <bool>true</bool>
   </property>
   <property name="geometry">
    <rect>
     <x>300</x>
     <y>440</y>
     <width>111</width>
     <height>41</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgba(255, 255, 255, 155);
font: 10pt &quot;黑体&quot;;</string>
   </property>
   <property name="text">
    <string>下一步</string>
   </property>
  </widget>
  <widget class="QPushButton" name="OverButton">
   <property name="enabled">
    <bool>true</bool>
   </property>
   <property name="geometry">
    <rect>
     <x>540</x>
     <y>490</y>
     <width>61</width>
     <height>41</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(255, 0, 0, 155);
color: rgb(255, 255, 255);
font: 10pt &quot;黑体&quot;;</string>
   </property>
   <property name="text">
    <string>退出</string>
   </property>
  </widget>
  <widget class="QRadioButton" name="radioButton">
   <property name="enabled">
    <bool>true</bool>
   </property>
   <property name="geometry">
    <rect>
     <x>230</x>
     <y>470</y>
     <width>71</width>
     <height>16</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">
font: 75 10pt &quot;Arial&quot;;</string>
   </property>
   <property name="text">
    <string>×1.0</string>
   </property>
  </widget>
  <widget class="QRadioButton" name="radioButton_2">
   <property name="enabled">
    <bool>true</bool>
   </property>
   <property name="geometry">
    <rect>
     <x>230</x>
     <y>490</y>
     <width>71</width>
     <height>16</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">
font: 75 10pt &quot;Arial&quot;;</string>
   </property>
   <property name="text">
    <string>×0.5</string>
   </property>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>70</x>
     <y>60</y>
     <width>91</width>
     <height>31</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 14pt &quot;Eras Bold ITC&quot;;</string>
   </property>
   <property name="text">
    <string>原数据：</string>
   </property>
  </widget>
  <widget class="QLabel" name="FE_num">
   <property name="geometry">
    <rect>
     <x>190</x>
     <y>60</y>
     <width>281</width>
     <height>31</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">color: rgb(0, 0, 255);
font: 75 14pt &quot;Agency FB&quot;;</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QPushButton" name="LastButton">
   <property name="enabled">
    <bool>true</bool>
   </property>
   <property name="geometry">
    <rect>
     <x>410</x>
     <y>440</y>
     <width>111</width>
     <height>41</height>
    </rect>
   </property>
   <property name="mouseTracking">
    <bool>false</bool>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgba(255, 255, 255, 155);
font: 10pt &quot;黑体&quot;;</string>
   </property>
   <property name="text">
    <string>上一步</string>
   </property>
  </widget>
  <widget class="QPushButton" name="StopButton">
   <property name="geometry">
    <rect>
     <x>410</x>
     <y>490</y>
     <width>111</width>
     <height>41</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgba(255, 255, 255, 155);
font: 10pt &quot;黑体&quot;;</string>
   </property>
   <property name="text">
    <string>停止</string>
   </property>
  </widget>
  <widget class="QRadioButton" name="radioButton_3">
   <property name="geometry">
    <rect>
     <x>230</x>
     <y>510</y>
     <width>71</width>
     <height>16</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">
font: 75 10pt &quot;Arial&quot;;</string>
   </property>
   <property name="text">
    <string>×2.0</string>
   </property>
  </widget>
  <widget class="QPushButton" name="ReButton">
   <property name="geometry">
    <rect>
     <x>540</x>
     <y>440</y>
     <width>61</width>
     <height>41</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">color: rgb(255, 255, 255);
background-color: rgb(0, 85, 255, 155);
font: 10pt &quot;黑体&quot;;</string>
   </property>
   <property name="text">
    <string>重置</string>
   </property>
  </widget>
  <widget class="QGroupBox" name="groupBox_3">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>180</y>
     <width>581</width>
     <height>191</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 10pt &quot;楷体&quot;;
border-bottom-color: rgb(0, 0, 0);
border-right-color: rgb(0, 0, 0);
background-color: rgb(255, 255, 255, 70);</string>
   </property>
   <property name="title">
    <string>过程可视化</string>
   </property>
   <widget class="QLabel" name="label_3">
    <property name="geometry">
     <rect>
      <x>480</x>
      <y>120</y>
      <width>41</width>
      <height>51</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 75 32pt &quot;Agency FB&quot;;
background-color: rgba(255, 255, 255, 0);</string>
    </property>
    <property name="text">
     <string>e：</string>
    </property>
   </widget>
   <widget class="QLCDNumber" name="lcdNumber">
    <property name="geometry">
     <rect>
      <x>470</x>
      <y>130</y>
      <width>101</width>
      <height>41</height>
     </rect>
    </property>
    <property name="segmentStyle">
     <enum>QLCDNumber::Flat</enum>
    </property>
   </widget>
   <widget class="QLabel" name="label_4">
    <property name="geometry">
     <rect>
      <x>480</x>
      <y>110</y>
      <width>20</width>
      <height>20</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgba(255, 255, 255, 0);</string>
    </property>
    <property name="text">
     <string>-1</string>
    </property>
   </widget>
   <widget class="QLabel" name="reslabel">
    <property name="geometry">
     <rect>
      <x>90</x>
      <y>120</y>
      <width>361</width>
      <height>41</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">text-decoration: underline;
color: rgb(255, 85, 0);
font: 16pt &quot;新宋体&quot;;
background-color: rgba(255, 255, 255, 0);</string>
    </property>
    <property name="text">
     <string>结果：</string>
    </property>
   </widget>
   <zorder>label_4</zorder>
   <zorder>lcdNumber</zorder>
   <zorder>label_3</zorder>
   <zorder>reslabel</zorder>
  </widget>
  <widget class="QLabel" name="Now_num">
   <property name="geometry">
    <rect>
     <x>190</x>
     <y>100</y>
     <width>281</width>
     <height>31</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">color: rgb(0, 0, 255);
font: 75 14pt &quot;Agency FB&quot;;</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>70</x>
     <y>100</y>
     <width>91</width>
     <height>31</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 14pt &quot;Eras Bold ITC&quot;;</string>
   </property>
   <property name="text">
    <string>现数据：</string>
   </property>
  </widget>
  <widget class="QQuickWidget" name="quickWidget">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>460</y>
     <width>141</width>
     <height>61</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgba(0, 255, 0, 0);</string>
   </property>
   <property name="resizeMode">
    <enum>QQuickWidget::SizeRootObjectToView</enum>
   </property>
   <property name="source">
    <url>
     <string>file:///D:/Qt/F-E_Conversion/Vi.qml</string>
    </url>
   </property>
  </widget>
  <widget class="QGroupBox" name="groupBox">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>30</y>
     <width>581</width>
     <height>111</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 10pt &quot;楷体&quot;;
background-color: rgba(255, 255, 255, 70);</string>
   </property>
   <property name="title">
    <string>数据变化</string>
   </property>
  </widget>
  <widget class="QGroupBox" name="groupBox_2">
   <property name="geometry">
    <rect>
     <x>620</x>
     <y>30</y>
     <width>341</width>
     <height>501</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 10pt &quot;楷体&quot;;
background-color: rgba(255, 255, 255, 70);</string>
   </property>
   <property name="title">
    <string>移动操作_代码</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_5">
   <property name="geometry">
    <rect>
     <x>300</x>
     <y>490</y>
     <width>111</width>
     <height>41</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">border-color: rgba(255, 255, 255, 55);
border-image: url(:/pic/0508f31a656101be8baa42eb294c74d8.gif);
background-color: rgba(255, 255, 255, 155);</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="label_6">
   <property name="geometry">
    <rect>
     <x>300</x>
     <y>530</y>
     <width>101</width>
     <height>16</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 75 8pt &quot;Agency FB&quot;;
color: rgb(0, 255, 0);</string>
   </property>
   <property name="text">
    <string>（自动进行中）</string>
   </property>
  </widget>
  <zorder>groupBox_2</zorder>
  <zorder>groupBox</zorder>
  <zorder>textBrowser</zorder>
  <zorder>AutoButton</zorder>
  <zorder>NextButton</zorder>
  <zorder>OverButton</zorder>
  <zorder>radioButton</zorder>
  <zorder>radioButton_2</zorder>
  <zorder>label</zorder>
  <zorder>FE_num</zorder>
  <zorder>LastButton</zorder>
  <zorder>StopButton</zorder>
  <zorder>radioButton_3</zorder>
  <zorder>ReButton</zorder>
  <zorder>groupBox_3</zorder>
  <zorder>Now_num</zorder>
  <zorder>label_2</zorder>
  <zorder>quickWidget</zorder>
  <zorder>label_5</zorder>
  <zorder>label_6</zorder>
 </widget>
 <customwidgets>
  <customwidget>
   <class>QQuickWidget</class>
   <extends>QWidget</extends>
   <header>QtQuickWidgets/QQuickWidget</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
